frontend/src/components/Main.vue

```diff
@@ -11,29 +11,12 @@
 <style scoped>
 .main-container {
     width: 100%;
-    height: 90vh;
+    height: auto;
     margin-top: 10vh;
     background-color: var(--background-color);
-    overflow: scroll;
-}
-
-.main-container::-webkit-scrollbar {
```

Comment:
I should revert this change and keep the styles for the scrollbar

Reply:
Okay